crates/codegen/language/definition/src/model/non_terminals/field.rs

```diff
@@ -15,6 +15,10 @@ pub struct FieldsErrorRecovery {
 pub struct FieldDelimiters {
     pub open: Identifier,
     pub close: Identifier,
+    /// How many tokens have to be matched to trigger the error recovery.
+    /// For ambiguous syntaxes this needs to be set to at least N, where N
+    /// is the token lookahead required to disambiguate the syntax.
+    pub tokens_matched_acceptance_threshold: Option<u8>,
```

Comment:
I suggest adding a comment here about what happens in case of `None`. IIUC, it will be similar to `u8::default()` (`0`), which means it will recover even if nothing matched?

Reply:
Glad you like it! Yes, the default will be 0 in our case, which is: always attempt recovering, even from "no" match for delimited bodies.

Done in https://github.com/NomicFoundation/slang/pull/862/commits/f41d7aa401e7393f84a45766fd6e8ec0cbe759aa